qubesmanager/qube_manager.py

```diff
@@ -103,13 +103,16 @@ def __init__(self):
                 "Halted" : QIcon(":/blank")
                 }
         self.outdatedIcons = {
+                "blocked" : QIcon(":/ban"),
                 "update" : QIcon(":/updateable"),
                 "outdated" : QIcon(":/outdated"),
                 "to-be-outdated" : QIcon(":/outdated"),
                 "eol": QIcon(':/warning'),
                 "skipped": QIcon(':/skipped')
                 }
         self.outdatedTooltips = {
+                "blocked" : self.tr(
+                    "The qube is prohibited from being started"),
```

Comment:
```suggestion
                    "The qube is prohibited from starting"),
```

(to avoid passive voice when it's not inconvenient)